contracts/staking/contracts/src/vaults/ZrxVault.sol

```diff
@@ -96,7 +96,7 @@ contract ZrxVault is
         _balances[owner] = _balances[owner].safeAdd(amount);
 
         // notify
-        emit ZrxDepositedIntoVault(msg.sender, owner, amount);
+        emit ZrxDepositedIntoVault(owner, amount);
```

Comment:
I prefer simple `Deposit` and `Withdrawal` events personally. We could also conceivably remove these events entirely and just use the ZRX `Transfer` events, though I guess there could be false positives.

Reply:
We could add a single `Deposit` event to prevent false positives. Such an event + the ZRX `Transfer` event should prevent false positives.